vendor/github.com/aws/aws-sdk-go/service/ec2/api.go

```diff
@@ -68339,6 +68339,24 @@ const (
 	// InstanceTypeR416xlarge is a InstanceType enum value
 	InstanceTypeR416xlarge = "r4.16xlarge"
 
+	// InstanceTypeX1Exlarge is a InstanceType enum value
```

Comment:
These duplicate existing variables :)

Also just an FYI, vendored modules like this one need to be updated via this process, rather than edited directly: https://github.com/kubernetes/kops/blob/18c4d3dac53a657ab898a08e04cf649956439021/docs/development/dependencies.md

Reply:
reverted